sendgrid/sendgrid.js

```diff
@@ -1,13 +1,16 @@
-export const sendEmailAdminDashboard = async (email) => {
+export const sendEmailAdminDashboard = async (email, application_id, company) => {
   const sgMail = require('@sendgrid/mail');
   sgMail.setApiKey(process.env.SENDGRID_API_KEY);
 
+  const url = process.env.NODE_ENV === "production" ? `https://florish-app-florish.vercel.app/reflection/${application_id}` : `http://localhost:3000/reflection/${application_id}`;
```

Comment:
would getting the hostname from `req.headers.host` in `send-email` and passing it to this function work? perhaps  an approach that would allow for less hardcoding.

Reply:
good call